File: multi line/sample.c
//be name khoda

#include <stdio.h>

int main()
{
	int ans=0,n,a;
	scanf("%d",&n);
	for(int i=0;i<n;i++)
	{
	    scanf("%d",&a);
	    ans+=a;
	}
	printf("%d\n",ans);
	return 0;
}

File: multi line/sample.java
//be name khoda
import java.util.Scanner;
public class Main {
    static Scanner sc = new Scanner(System.in);
    public static void main(String[] args) {
        int ans=0;
        int n = sc.nextInt();
        for(int i=0;i<n;i++) {
            int a = sc.nextInt();
            ans = ans + a;
        }
        System.out.println(ans);
    }
}
